butterknife/src/main/java/butterknife/internal/ViewInjector.java

```diff
@@ -129,7 +129,9 @@ private void emitCollectionBinding(StringBuilder builder, CollectionBinding bind
         builder.append(',');
       }
       builder.append("\n        ");
-      emitCastIfNeeded(builder, binding.getType());
+      if (!binding.getType().equals(VIEW_TYPE)) {
```

Comment:
Can we extract this onto `binding` as a `requiresCast()` method? I don't like the duplicated logic all over.